tools/lldb-moduleimport-test/lldb-moduleimport-test.cpp

```diff
@@ -181,30 +181,38 @@ int main(int argc, char **argv) {
   INITIALIZE_LLVM();
 
   // Command line handling.
-  llvm::cl::list<std::string> InputNames(
-    llvm::cl::Positional, llvm::cl::desc("compiled_swift_file1.o ..."),
-    llvm::cl::OneOrMore);
+  using namespace llvm::cl;
+  static OptionCategory Visible("Specific Options");
+  HideUnrelatedOptions({&Visible});
```

Comment:
It looks like `HideUnrelatedOptions(None)` will hide everything, and then the following options don't need to get a category.

Reply:
Good to know! But this way it also prints nicely in the --help output as

General options:
  --version ...
Specific options:
  --dump-module ...

Reply:
By the way, do you happen to know what's up with the failure in https://ci.swift.org/job/swift-PR-osx/8961/ ?

Reply:
@graydon's disabling the test in #20581 while he investigates.